spring-boot/autoconfigure/pom.xml

```diff
@@ -4,16 +4,16 @@
 
   <parent>
     <groupId>org.togglz</groupId>
-    <artifactId>togglz-project</artifactId>
+    <artifactId>togglz-spring-boot</artifactId>
     <version>2.6.0-SNAPSHOT</version>
   </parent>
 
-  <artifactId>togglz-spring-boot-starter</artifactId>
-  <name>Togglz - Spring Boot Starter</name>
-  <description>Togglz - Spring Boot Starter</description>
+  <artifactId>togglz-spring-boot-autoconfigure</artifactId>
+  <name>Togglz - Spring Boot Autoconfiguration</name>
+  <description>Togglz - Spring Boot Autoconfiguration </description>
```

Comment:
Could be something like, Dependency to enable Togglz Autoconfiguration in Spring Boot environment.